py3status/modules/sysdata.py

```diff
@@ -277,6 +328,16 @@ def _get_cputemp(self, zone, unit):
     def sysdata(self):
         sys = {"max_used_percent": 0, "temp_unit": self.temp_unit}
 
+        if self.init["cpu_freq_max"] or self.init["cpu_freq_avg"]:
+            cpu_freqs = self._calc_cpu_freqs(self.cpu_freq_unit)
+            if self.init["cpu_freq_max"]:
+                sys["cpu_freq_max"] = cpu_freqs["max"]
+                self.py3.threshold_get_color(sys["cpu_freq_max"], "cpu")
```

Comment:
Oh! So you meant just doing this?

```suggestion
                self.py3.threshold_get_color(sys["cpu_freq_max"], "cpu_freq")
```

Reply:
Yes! sorry for the confusion :)